<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DensityDockWidgetBase</class>
 <widget class="QDockWidget" name="DensityDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>453</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Density Analysis</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Select original data layer</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QgsMapLayerComboBox" name="dataComboBox"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Select density heatmap polygon layer</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QgsMapLayerComboBox" name="densityHeatmapComboBox"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Density heatmap attributes</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Score</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QgsFieldComboBox" name="idComboBox"/>
      </item>
      <item row="1" column="1">
       <widget class="QgsFieldComboBox" name="countComboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QComboBox" name="zoomComboBox">
        <item>
         <property name="text">
          <string>No action</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Auto pan</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Auto zoom</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="applyButton">
        <property name="text">
         <string>Display Density Values</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="clearButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="resultsTable"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsfieldcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
